ffi/src/lib.rs

```diff
@@ -16,10 +16,40 @@ use delta_kernel::snapshot::Snapshot;
 use delta_kernel::{DeltaResult, Engine, Error};
 
 mod handle;
-use handle::{ArcHandle, SizedArcHandle, Unconstructable};
+use handle::{False, Handle, HandleDescriptor, True, Unconstructable};
 
 pub mod scan;
 
+#[repr(C)]
+pub struct Foo {
```

Comment:
this is test code right? we probably don't want it in here?

Reply:
Good point. Deleted this and a bunch of other junk